Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -1013,19 +1018,36 @@ extension CarPlayManager: CPMapTemplateDelegate {
         navigationMapView.mapView.mapboxMap.setCamera(to: cameraOptions)
     }
 
-    private func popToRootTemplate(interfaceController: CPInterfaceController?, animated: Bool) {
+    private func popToRootTemplate(interfaceController: CPInterfaceController?,
+                                   animated: Bool,
+                                   completion: ((Bool, Error?) -> Void)? = nil) {
         guard let interfaceController = interfaceController,
               interfaceController.templates.count > 1 else { return }
         
         if #available(iOS 14.0, *) {
-            interfaceController.popToRootTemplate(animated: animated) { _, _ in
-                // No-op
-            }
+            interfaceController.popToRootTemplate(animated: animated, completion: completion)
         } else {
             interfaceController.popToRootTemplate(animated: animated)
+            completion?(true, nil)
```

Comment:
Unfortunately I cannot verify this fix on iOS 13. I assume that current implementation is equivalent to what we had before and since there are less and less iOS 12/13 users each day, I think we could leave it as is

Reply:
I've checked on iOS 13.7 simulator and this works fine, so there's no need for modifications